migrations/emit-complete.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path'),
+    db = require('../db');
+
+module.exports = {
+  name: 'emit-complete',
+  created: new Date(2017, 2, 27),
+  run: function(callback) {
+    db.getSettings(function(err, data) {
+      if (err) {
+        return callback(err);
+      }
+      var rules = data.settings.tasks && data.settings.tasks.rules;
+      if (!rules) {
+        // no rules configured
+        return callback();
+      }
+
+      if (rules.indexOf('_complete') !== -1) {
+        // complete task already configured
+        return callback();
+      }
+
+      rules = rules.replace(/(.*)\}[ \n\t]*\}[ \n\t]*$/, '$1 emit("_complete", { _id: true });}}');
```

Comment:
Regex of doom. Can we get a comment with examples of what this catches, and what it gets turned into with the replace?

(will still merge, do it in master).